lib/packwerk/rails_load_paths.rb

```diff
@@ -2,22 +2,26 @@
 # frozen_string_literal: true
 
 require "bundler"
+gem "railties"
```

Comment:
We should constrain this to the version that adds `Rails.autoloaders` with Zeitwerk by default.

Reply:
I think this should be the version where Rails removed the classic autoloader. So that's 7.0: https://guides.rubyonrails.org/classic_to_zeitwerk_howto.html#applications-running-rails-7